### PR TITLE
added nov2 file

### PR DESCRIPTION
**Pull request by sindhams**

### CODE REVIEW

**Review by SameeraGrandhi — APPROVED** [review]

**Review by madhaviai — APPROVED** [review]
just can add review

**Review by DeepikaChary — CHANGES_REQUESTED** [review]
Looks ok

File: nov2 [hunk 1]
```diff
@@ -0,0 +1 @@
+nov2 feature
```
Comment (DeepikaChary) [inline nov2]: Looks like u need to change the logic

**Review by DeepikaChary — CHANGES_REQUESTED** [review]

File: nov2 (hunk 1 above)
Comment (madhaviai) [reply nov2]: yes

Comment (sindhams) [reply nov2]: ok

**Review by sindhams — COMMENTED** [review]
as per new changes

**Review by DeepikaChary — APPROVED** [review]

**Pull request merged**